Update to Supported USB Sticks

Aeotec-ccheng opened the pull request:
Updated Aeotec Z-Stick Gen5 name, and add Z-Stick 7 to list of compatible hardware.

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Aeotec-ccheng committed:
Update to Supported USB Sticks

Updated Aeotec Z-Stick Gen5 name, and add Z-Stick 7 to list of compatible hardware.

Aeotec-ccheng committed:
Adding Z-Pi 7

Added Z-Pi 7 to the list

frenck commented on source/_docs/z-wave/controllers.markdown:
```diff
@@ -13,7 +13,9 @@ This Z-Wave integration is deprecated and replaced with a [new implementation ba
 
-- Aeotec Z-Stick Series 5 (see note below)
+- Aeotec Z-Stick Gen5 (see note below)
+- Aeotec Z-Stick 7
+- Aeotec Z-Pi 7
```
Are these compatible with OpenZWave as well?

Aeotec-ccheng replied:
Z-Stick 7 and Z-Pi7  are both bridge controllers, they won't work with OpenZWave.

frenck replied:
Well this page is for OpenZWave, so that makes this change invalid.

Aeotec-ccheng replied:
Got it, thank you. Didn't realize that since it mentions info about Bridge controllers not being for OpenZWave. I'll close this one up.

Aeotec-ccheng replied:
@frenck - The ZST10 is a Bridge controller as well which is why we added the Z-Stick 7 and Z-Pi 7 to this list: https://products.z-wavealliance.org/products/4260?selectedFrequencyId=-1

Aeotec-ccheng committed:
Remove all bridge Controllers

Removed all Bridge controllers utilizing Series 700.

Z-Stick 7
Z-Pi 7
Zooz ZST10

Aeotec-ccheng commented:
Error in list of supported products. ZST10 is a bridge controller and would not be supported by OpenZWave

frenck reviewed (approved):
Thanks, @Aeotec-ccheng 👍
